esy-lib/posix_getumask_patch.c

```diff
@@ -0,0 +1,45 @@
+#include <caml/mlvalues.h>
+#include <caml/memory.h>
+#include <caml/alloc.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+
+#if __APPLE__ || __linux__
+#else 
+#include <io.h>
+#endif
+
+
+CAMLprim value
+esy_getumask(value unit) {
+    CAMLparam1( unit );
```

Comment:
`CAMLparam0()` is more appropriate here.

Reply:
Done